asapdiscovery-ml/asapdiscovery/ml/utils.py

```diff
@@ -10,12 +10,17 @@ def build_dataset(
     in_files,
     model_type,
     exp_fn,
+    xtal_pat=r"[^/]+$",
```

Comment:
Same here I would do the filtering outside the function by whatever means nessecary and then accept iterables of the relevant data, that way regex is not absolutely required.